src/index.html

```diff
@@ -10,23 +10,39 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
+      <label>
+        <input
+          class="search-bar search-bar__big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="small"
+    >
+      <label>
+        <input
+          class="search-bar search-bar__small"
```

Comment:
Similar to the previous comment, 'search-bar__small' should be replaced with a class name that reflects the function or context of the element rather than its size.